modules/sdk-coin-tia/src/tia.ts

```diff
@@ -0,0 +1,205 @@
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+
+    if (!staticsCoin) {
+      throw new Error('missing required constructor parameter staticsCoin');
+    }
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Tia(bitgo, staticsCoin);
+  }
+
+  getBuilder(): TransactionBuilderFactory {
+    return new TransactionBuilderFactory(coins.get(this.getChain()));
+  }
+
+  /** @inheritDoc **/
+  isValidAddress(address: string): boolean {
+    return utils.isValidAddress(address) || utils.isValidValidatorAddress(address);
+  }
+
+  /** @inheritDoc **/
+  async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
+    let totalAmount = new BigNumber(0);
+    const coinConfig = coins.get(this.getChain());
+    const { txPrebuild, txParams } = params;
+    const rawTx = txPrebuild.txHex;
+    if (!rawTx) {
+      throw new Error('missing required tx prebuild property txHex');
+    }
+    const transaction = await new TransactionBuilderFactory(coinConfig).from(rawTx).build();
+    const explainedTx = transaction.explainTransaction();
+
+    if (txParams.recipients && txParams.recipients.length > 0) {
+      const filteredRecipients = txParams.recipients?.map((recipient) => _.pick(recipient, ['address', 'amount']));
+      const filteredOutputs = explainedTx.outputs.map((output) => _.pick(output, ['address', 'amount']));
+
+      if (!_.isEqual(filteredOutputs, filteredRecipients)) {
+        throw new Error('Tx outputs does not match with expected txParams recipients');
+      }
+      // WithdrawDelegatorRewards transaction doesn't have amount
+      if (transaction.type !== TransactionType.StakingWithdraw) {
+        for (const recipients of txParams.recipients) {
+          totalAmount = totalAmount.plus(recipients.amount);
+        }
+        if (!totalAmount.isEqualTo(explainedTx.outputAmount)) {
+          throw new Error('Tx total amount does not match with expected total amount field');
+        }
+      }
+    }
+    return true;
+  }
+
+  /** @inheritDoc **/
+  async explainTransaction(options: { txHex: string }): Promise<ExplanationResult> {
+    if (!options.txHex) {
+      throw new Error('missing required txHex parameter');
+    }
+    try {
+      const transactionBuilder = new TransactionBuilderFactory(coins.get(this.getChain())).from(options.txHex);
+      const transaction = await transactionBuilder.build();
+      return transaction.explainTransaction();
+    } catch (e) {
+      throw new Error('Invalid transaction: ' + e.message);
+    }
+  }
+
+  /**
+   * Sign a transaction with a single private key
+   * @param params parameters in the form of { txPrebuild: {txHex}, prv }
+   * @returns signed transaction in the form of { txHex }
+   */
+  async signTransaction(
```

Comment:
Same idea as previous files: most methods here seem very similar/common and could exist in a common class